### PR TITLE
Add support for zstd compression

### PR DESCRIPTION
**Pull request by kontura**

### CODE REVIEW

**Comment by kontura** [comment]
Added conditional libzstd dependency to specfile and a new commit with cmake project language.

File: drpm.spec [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Do not build with zstd for RHEL >= 8
+%if 0%{?rhel} && 0%{?rhel} >= 8
```
Comment (dmach) [inline drpm.spec]: keep only this part: %if 0%{?rhel}

File: drpm.spec [hunk 2]
```diff
@@ -42,13 +52,14 @@ The drpm-devel package provides a C interface (drpm.h) for the drpm library.
 %autosetup
 mkdir build
 
+%global _cmake_opts \\\
```
Comment (dmach) [inline drpm.spec]: remove _cmake_opts macro and move it's contents directly on the %cmake ..  line

File: CMakeLists.txt [hunk 3]
```diff
@@ -11,6 +12,9 @@ find_package(LibLZMA REQUIRED)
 find_package(PkgConfig REQUIRED)
 pkg_check_modules(RPM rpm REQUIRED)
 pkg_check_modules(LIBCRYPTO libcrypto REQUIRED)
+if (WITH_ZSTD)
```
Comment (dmach) [inline CMakeLists.txt]: remove the space after if: if(...) for consistency with endif()

**Comment by kontura** [comment]
I have made the requested changes.

**Pull request merged**